src/views/RequestDocument.vue

```diff
@@ -0,0 +1,295 @@
+<template>
```

Comment:
Sounds that this change is out of context of this PR.

Reply:
fix lint error

Reply:
This is a new file, isn't a lint fix.